dashboard/app/views/peer_reviews/index.html.haml

```diff
@@ -0,0 +1,33 @@
+%h1 Listing peer_reviews
```

Comment:
I think this will eventually list all my active peer reviews, by script.  But it could also make sense as the instructor overview?

Either way this probably won't be the main entry point into peer reviews — it will be somewhere inline with the course.